.travis.yml

```diff
@@ -79,6 +79,15 @@ before_install:
     - which python; python --version
     - tools/build_versions.py
     - tools/check_sdist.py
+    # It is really annoying if flake8 causes your tests to fail first
+    # but in the event that you have a typo, flake8 is pretty good at
+    # catching it.
+    # Therefore, it is best to run flake8 first, on a single build
+    # with the oldest version of python we support
+    - if [[ $TEST_FLAKE8 == "1" ]]; then
+        pip install flake8;
+        flake8;
```

Comment:
You probably need to keep the same flake8 arguments as before.

Reply:
I think it is all in the `setup.cfg`. I added a dummy commit to trip up flake8 to test it.